### PR TITLE
Update Somali UI text

### PR DESCRIPTION
**Pull request by elisekalstad**
### Description
Added Somali UI text

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: server/conf/messages.so [hunk 1]
```diff
@@ -5,3 +5,325 @@
...
+guest=Guest
+
+# Label read by screen readers for the language dropdown shown in the page header.
+label.languageSr=Dooro luqad
+
+# Message with user's name to show who you are logged in as.
+header.userName=U soo gal sidii {0}
+
+# Error message to answer a required question
+validation.isRequired=Su'aashan ayaa loo baahan yahay.
```
Comment (gwendolyngoetz) [inline server/conf/messages.so:35]: Add a second apostrophe `Su'aashan` > `Su''aashan`

File: server/conf/messages.so [hunk 2]
```diff
@@ -5,3 +5,325 @@
...
+# APPLICANT HOME PAGE - contains text specific to the applicant's home page. #
+#----------------------------------------------------------------------------#
+
+# The text on the button an applicant clicks to apply to a specific program.
+button.apply=Codso
+# The text read for screen readers.
+button.applySr=Ka Codso {0}
+
+# "Get benefits" floating text
+content.benefits=Hel faa'iidooyin
```
Comment (gwendolyngoetz) [inline server/conf/messages.so:116]: Needs a second apostrophe

File: server/conf/messages.so [hunk 3]
```diff
@@ -5,3 +5,325 @@
...
+
+# Long form description of the CiviForm site shown to the applicant. Line 1.
+content.description1=CiviForm wuxuu kuu ogolaanayaa inaad codsato faa'iidooyin badan hal mar adiga oo dib u isticmaalaya macluumaadka.
```
Comment (gwendolyngoetz) [inline server/conf/messages.so:119]: Needs a second apostrophe

File: server/conf/messages.so [hunk 4]
```diff
@@ -5,3 +5,325 @@
...
+validation.textTooLong=Waa in uu ka kooban yahay ugu badnaan {0} xaraf.
+validation.textTooShort=Waa in uu ka kooban yahay ugu yaraan {0} xaraf.
+
+
+#---------------------------------------------------------------------#
+# MULTI OPTION QUESTION ADMIN EDIT - text specific when creating/editing a multi option question. #
+#---------------------------------------------------------------------#
+
+# Validation errors that appear if an admin leaves a multi option question blank.
+adminValidation.multiOptionEmpty=Su'aalaha xulashada badan ma yeelan karaan ikhtiyaaro madhan.
```
Comment (gwendolyngoetz) [inline server/conf/messages.so:329]: Needs a second apostrophe

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]
@elisekalstad there are a few spots that need to have the double apostrophe

**Review by bion — APPROVED** [review]

**Pull request merged**